Project.toml

```
@@ -14,10 +14,10 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 [compat]
 FileIO = "1.2.3"
 ImageCore = "0.8.1, 0.9"
-ProtoBuf = "0.10, 0.11"
+ProtoBuf = "0.10, 0.11, 1.0.11"
```

Do we actually still support those old versions of ProtoBuf?
or just

```suggestion
ProtoBuf = "1.0.11"
```

---

Nope lol